test/e2e/apimachinery/validatingadmissionpolicy.go

```diff
@@ -36,7 +36,7 @@ import (
 	admissionapi "k8s.io/pod-security-admission/api"
 )
 
-var _ = SIGDescribe("ValidatingAdmissionPolicy [Privileged:ClusterAdmin][Alpha]", feature.ValidatingAdmissionPolicy, func() {
+var _ = SIGDescribe("ValidatingAdmissionPolicy [Privileged:ClusterAdmin]", feature.ValidatingAdmissionPolicy, func() {
```

Comment:
Yes it can be removed. The tag is to filter out all alpha features in job `pull-kubernetes-e2e-kind-alpha-features`. Automatic stability tagging is better than manual for sure. I will remember to update the job once this PR merges.

Reply:
The feature gate is `feature.ValidatingAdmissionPolicy`. I see you've already added `feature.ValidatingAdmissionPolicy` as the optional second parameter. That's a great change.

Reply:
There's a slight difference between `Feature` and `FeatureGate`: only the latter have stability levels and eventually get removed when the feature reaches GA, whereas `Feature` is a permanent tag that identifies some functionality. The pending KEP tries to clarify that.

I am going to use `FeatureGate`, so the test now is:
```
    [sig-api-machinery] ValidatingAdmissionPolicy [Privileged:ClusterAdmin] [FeatureGate:ValidatingAdmissionPolicy] [Alpha] should validate against a Deployment
```

This doesn't break the job.